exporter/trace/cloudtrace.go

```diff
@@ -233,26 +233,23 @@ type Exporter struct {
 }
 
 // InstallNewPipeline instantiates a NewExportPipeline and registers it globally.
-func InstallNewPipeline(opts []Option, topts ...sdktrace.ProviderOption) (apitrace.Provider, func(), error) {
+func InstallNewPipeline(opts []Option, topts ...sdktrace.TracerProviderOption) (apitrace.TracerProvider, func(), error) {
 	tp, flush, err := NewExportPipeline(opts, topts...)
 	if err != nil {
 		return nil, nil, err
 	}
-	global.SetTraceProvider(tp)
+	global.SetTracerProvider(tp)
 	return tp, flush, err
 }
 
 // NewExportPipeline sets up a complete export pipeline with the recommended setup
 // for trace provider. Returns provider, flush function, and errors.
-func NewExportPipeline(opts []Option, topts ...sdktrace.ProviderOption) (apitrace.Provider, func(), error) {
+func NewExportPipeline(opts []Option, topts ...sdktrace.TracerProviderOption) (apitrace.TracerProvider, func(), error) {
 	exporter, err := NewExporter(opts...)
 	if err != nil {
 		return nil, nil, err
 	}
-	tp, err := sdktrace.NewProvider(append(topts, sdktrace.WithSyncer(exporter))...)
-	if err != nil {
-		return nil, nil, err
-	}
+	tp := sdktrace.NewTracerProvider(append(topts, sdktrace.WithSyncer(exporter))...)
 	return tp, exporter.traceExporter.Flush, nil
```

Comment:
Would this be somewhat cleaner if we returned `exporter.Shutdown` here instead or `exporter.traceExporter.Flush`?